emcc.py

```diff
@@ -391,7 +391,8 @@ def run():
 
   if DEBUG:
     logger.warning('invocation: ' + ' '.join(sys.argv) + (' + ' + EMCC_CFLAGS if EMCC_CFLAGS else '') + '  (in ' + os.getcwd() + ')')
-  if EMCC_CFLAGS:
+  # Not doing a compilation if we have '-v', so don't use extra flags
```

Comment:
reading the code later down, that comment is not accurate - we may do a compilation even with `-v`. And that seems like the right behavior since gcc does it too.

I think the right solution here might be to append EMCC_CFLAGS if and only if we are doing a compilation. That means after we check for `-v` with no other arguments, and other options that halt us before compilation, so around like 492 (right before the configure handling code, starting with `# If this is a configure-type thing`).